.gitignore

```diff
@@ -0,0 +1,149 @@
+
+# Created by https://www.gitignore.io/api/java,maven,windows,intellij
```

Comment:
If this is only getting added now then files which should never have been commited will be in the repo. The git cache will need to performed to remove these files.